sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/ActionDescriptorFactory.scala

```diff
@@ -48,11 +50,23 @@ private[impl] object ActionDescriptorFactory extends ComponentDescriptorFactory
           .withKalixOptions(kalixOptions)
       }
 
+    val subscriptionTopicMethods = component.getMethods
+      .filter(hasTopicSubscription)
+      .sorted // make sure we get the methods in deterministic order
+      .map { method =>
+        val subscriptionOptions = eventingInForTopic(method)
+        val kalixOptions =
+          kalix.MethodOptions.newBuilder().setEventing(subscriptionOptions).build()
+
+        KalixMethod(RestServiceMethod(method))
+          .withKalixOptions(kalixOptions)
+      }
+
     val serviceName = nameGenerator.getName(component.getSimpleName)
     ComponentDescriptor(
       nameGenerator,
       serviceName,
       component.getPackageName,
-      springAnnotatedMethods ++ subscriptionMethods)
+      springAnnotatedMethods ++ subscriptionMethods ++ subscriptionTopicMethods)
```

Comment:
I should change `subscriptionMethods` for `subscriptionValueEntityMethods`.

Reply:
yep, I think it will increase clarity